android/titanium/src/java/org/appcelerator/titanium/TiBlob.java

```diff
@@ -872,6 +872,60 @@ public TiBlob imageAsCompressed(Number compressionQuality)
 		return result;
 	}
 
+	@Kroll.method
+	public TiBlob imageAsWebP(Number compressionQuality)
```

Comment:
```suggestion
	public TiBlob imageAsWebP(@Kroll.argument(optional = true) Number compressionQuality)
```

Reply:
Additionally, though our current APIs are sync around image cropping/modifications, the Android docs suggest that the compression/conversion of a bitmap can be quite slow. I think this method should explicitly be async (i.e. take a callback and/or return a Promise)